src/api/webrpc/gatewayer_mock_test.go

```diff
@@ -26,6 +26,31 @@ func NewGatewayerMock() *GatewayerMock {
 	return &GatewayerMock{}
 }
 
+// GetInputData returns the data of an inputs
+func (m *GatewayerMock) GetInputData(in cipher.SHA256) (*historydb.UxOut, error) {
+	return nil, nil
+}
+
+// GetTransactionInputsData returns the inputs data of a transaction
+func (m *GatewayerMock) GetTransactionInputsData(tx *coin.Transaction) ([]*historydb.UxOut, error) {
+	return nil, nil
+}
+
+// GetBlockInputsData returns the inputs data of a block
+func (m *GatewayerMock) GetBlockInputsData(block *coin.Block) ([][]*historydb.UxOut, error) {
+	return nil, nil
+}
+
+// GetSignedBlockInputsData returns the inputs data of a signed block
+func (m *GatewayerMock) GetSignedBlockInputsData(block *coin.SignedBlock) ([][]*historydb.UxOut, error) {
+	return nil, nil
+}
+
+// GetSignedBlocksInputsData returns the inputs data of a signed blocks slice
+func (m *GatewayerMock) GetSignedBlocksInputsData(blocks []coin.SignedBlock) ([][][]*historydb.UxOut, error) {
+	return nil, nil
+}
+
```

Comment:
The `gatewayer_mock_test.go` is generated by `goautomock`, do not add code there,  otherwise the code will be wiped after running the goautomock.  run `go install`, then `go generate` to automaticlly generate mocker for new methods in Gatewayer interface. check the code in gui/transaction_test.go to see how to use the generated mocker.